src/features/cards/organisms/card-item.js

```diff
@@ -29,29 +83,115 @@ CardItem.propTypes = {
   onUsefulClick: PropTypes.func.isRequired,
 }
 
-const CardHeading = ({ card, onUsefulClick }) => (
+const CardFlagWithNumber = ({ usefulFor, isUseful, onUsefulClick }) => (
+  <div>
+    <FlagToggler onUsefulClick={onUsefulClick} isUseful={isUseful} />
+    <CenterHorizontal>
+      <Link to="/">{usefulFor}</Link>
+    </CenterHorizontal>
+  </div>
+)
+
+CardFlagWithNumber.propTypes = {
+  onUsefulClick: PropTypes.func.isRequired,
+  usefulFor: PropTypes.number.isRequired,
+  isUseful: PropTypes.bool.isRequired,
+}
+
+const FlagToggler = ({ isUseful, onUsefulClick }) => {
+  // const [isUseful, setIsUseful] = useState(usefulFor)
+
+  return (
+    <IconWrapper
+      onClick={() => {
+        // setIsUseful(!isUseful)
+        onUsefulClick()
+      }}
+    >
+      {isUseful ? (
+        <Icon name="bookmark-solid" fill="red" />
+      ) : (
+        <Icon name="bookmark-regular" />
+      )}
+    </IconWrapper>
+  )
+}
+
+FlagToggler.propTypes = {
+  onUsefulClick: PropTypes.func.isRequired,
+  isUseful: PropTypes.bool.isRequired,
+}
+
+const CardHeader = ({ card }) => (
   <HeadingLine>
     <Link to={`/open/${card.id}`}>
-      <H3 narrow>{card.title}</H3>
+      <H2 narrow>{card.title}</H2>
     </Link>
-    <Row basis="25%" justify="space-between">
-      {card.meta.isUseful ? (
-        <ButtonPrimary small title="Remove from saved" onClick={onUsefulClick}>
-          Saved
-        </ButtonPrimary>
-      ) : (
-        <Button small title="Add to saved" onClick={onUsefulClick}>
-          Save
-        </Button>
-      )}
-      {card.usefulFor > 0 ? <div>{card.usefulFor}</div> : <div>&nbsp;</div>}
+    <Row basis="25%" justify="flex-end" gap="1.4em" align="center">
       {card.meta.canEdit && <Link to={`/edit/${card.id}`}>Edit</Link>}
-      <i>{format(new Date(card.createdAt), "HH:MM MM/DD/YYYY")}</i>
+      <PopUpDelete title={card.title} />
+      <Icon name="dots-v" height="1.6rem" />
     </Row>
   </HeadingLine>
 )
+const CardInfo = (card) => (
+  <Row>
+    <Text small>
+      {" "}
+      {format(new Date(card.createdAt), "HH:MM MM/DD/YYYY")} by Author
+    </Text>
+  </Row>
+)
+
+const HeadingLine = styled.div`
+  display: flex;
+  flex-flow: row nowrap;
+  justify-content: space-between;
+  flex-shrink: 0;
+  line-height: 2.4rem;
+`
+const IconWrapper = styled.div`
+  padding: 0 0.6rem;
+  }
+  &:disabled {
+    opacity: 0.3;
+  }
+  
+  & #icon:hover {
```

Comment:
Please, don't use id selectors.

Reply:
I am going to fix/change work with svg in another branch/pr.

Reply:
Maybe fix in that?